tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -59,14 +60,14 @@ func NodePoolSetup(client *rancher.Client, nodeRoles []NodeRoles, ClusterID, Nod
 
 // ScaleWorkerNodePool is a helper method that will add a worker node pool to the existing RKE1 cluster. Once done, it will scale
 // the worker node pool to add a worker node, scale it back down to remove the worker node, and then delete the worker node pool.
-func ScaleWorkerNodePool(client *rancher.Client, nodeRoles []NodeRoles, ClusterID, NodeTemplateID string) error {
+func ScaleNewWorkerNodePool(client *rancher.Client, nodeRoles []NodeRoles, cluster *management.Cluster, nodeTemplate *nodetemplates.NodeTemplate) error {
```

Comment:
why can't we do something similar to the machine pools func

Reply:
Updated from Daniel's PR